pkg/kubelet/cri/remote/remote_image_test.go

```diff
@@ -0,0 +1,103 @@
+/*
+Copyright 2017 The Kubernetes Authors.
```

Comment:
nit: fix the year

Reply:
fixed

Reply:
@SergeyKanzhelev other than this, does it look good?

Reply:
sorry dropped the ball on this =). Looking